test_app/after/src/others.erl

```diff
@@ -44,6 +44,27 @@ receive_expr() ->
     receive
         with ->
             {no, timeout}
+    end,
+    receive after 1000 ->
+        {only, timeout}
+    end,
```

Comment:
@juanbono @diegomanuel: Do you prefer this to be formatted this way or is it better as follows?
```erlang
receive
after 1000 ->
    {only, timeout}
end,
```

Reply:
I'd go with what is in `others.erl`. The other alternatives "look like" something's missing.

Reply:
To be clear:
```erlang
receive after 1000 ->
    {only, timeout}
end
```

Reply:
I'm with Paulo, the first alternative looks nicer to me:
```erlang
receive
after 1000 ->
    {only, timeout}
end,
```